eslint.config.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const neo = require('neostandard')
+
+module.exports = [
+  ...neo({
+    env: ['jest'],
+    ignores: [
+      'lib/llhttp',
+      'test/fixtures/wpt',
+      'node_modules',
+      'undici-fetch.js'
+    ],
+    ts: true
+  }),
+  {
+    rules: {
+      '@stylistic/comma-dangle': ['error', {
+        arrays: 'never',
+        objects: 'never',
+        imports: 'never',
+        exports: 'never',
+        functions: 'never'
+      }],
+      '@typescript-eslint/no-redeclare': 'off',
```

Comment:
Was this a particular issue with TS-files? Maybe it's something we should tweak in neostandard itself?

Reply:
this basically happens if you have something like


```ts
export interface CacheStorage {
  match (request: RequestInfo, options?: MultiCacheQueryOptions): Promise<Response | undefined>,
  has (cacheName: string): Promise<boolean>,
  open (cacheName: string): Promise<Cache>,
  delete (cacheName: string): Promise<boolean>,
  keys (): Promise<string[]>
}

declare const CacheStorage: {
  prototype: CacheStorage
  new(): CacheStorage
}
```

Because the interface CacheStorage exists declaring the const CacheStorage results in a no-redeclare linting error